### PR TITLE
Add test procedure doc for images

### PR DESCRIPTION
**Pull request by coryrc**
Document some tribal knowledge.

### CODE REVIEW

File: images/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Images README
+
+## Testing Images
+
+Images are tested in production.
```
Comment (spiffxp) [inline images/README.md:5]:
```suggestion
There is no automated testing pipeline for images:
- Any jobs that use the `:latest` tag use the latest published image immediately
- Any jobs that use a `:v{date}-{sha}[-{variant}]` tag (e.g. `:v20200407-c818676-master`) are updated to use the latest published image ~daily.  This is accomplished by PR's created by the [autobumper prowjob](https://testgrid.k8s.io/sig-testing-prow#autobump-prow), which are merged by [test-infra oncall](go.k8s.io/oncall) once a day during weekdays.
```

File: images/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+1. Push a PR changing something in the image directory, preferably not near the time the Prow auto-bump occurs (otherwise, it will hard to figure out the true failure reason).
+
+1. Grep the [prow configs](../config/jobs) to find out which jobs are using <image-name>:latest and monitor [the TestGrid](http://testgrid.k8s.io) for new failures corresponding to your change.
```
Comment (spiffxp) [inline images/README.md:9]:
```suggestion
1. Grep the [prowjob configs](/config/jobs) to find out which jobs are using gcr.io/k8s-testimages/<image-name>:latest and monitor [TestGrid](http://testgrid.k8s.io) for new failures corresponding to your change.
```

File: images/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+    * On failure, send a new PR to rollback your last one or a fix if you know immediately.
+    * Some of these images might be presubmits; you could monitor them at http://prow.k8s.io
+
+1. You are done. If more breaks happen later, the oncall will take care of it.
```
Comment (spiffxp) [inline images/README.md:14]:
```suggestion
1. You are done. If more breaks happen later, [test-infra oncall](go.k8s.io/oncall) will take care of it.
```

File: images/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Images README
+
+## Testing Images
+
+Images are tested in production.
+
+1. Push a PR changing something in the image directory, preferably not near the time the Prow auto-bump occurs (otherwise, it will hard to figure out the true failure reason).
```
[the remark this answers is not in the document]
Comment (coryrc) [reply images/README.md]: I changed Push->Merge and removed other suggestion.

File: images/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+## Testing Images
+
+There is no automated testing pipeline for images:
+- Any jobs that use the `:latest` tag use the latest published image immediately
+- Any jobs that use a `:v{date}-{sha}[-{variant}]` tag (e.g. `:v20200407-c818676-master`) are updated to use the latest published image ~daily.  This is accomplished by PR's created by the [autobumper prowjob](https://testgrid.k8s.io/sig-testing-prow#autobump-prow), which are merged by [test-infra oncall](go.k8s.io/oncall) once a day during weekdays.
+
+1. Merge a PR changing something in the image directory.
+
+1. Grep the [prowjob configs](/config/jobs) to find out which jobs are using gcr.io/k8s-testimages/<image-name>:latest and monitor [TestGrid](http://testgrid.k8s.io) for new failures corresponding to your change.
```
Comment (spiffxp) [inline images/README.md:13]: Sorry I didn't catch this earlier, this renders as gcr.io/k8s-testimages/:latest
```suggestion
1. Grep the [prowjob configs](/config/jobs) to find out which jobs are using `gcr.io/k8s-testimages/<image-name>:latest` and monitor [TestGrid](http://testgrid.k8s.io) for new failures corresponding to your change.
```

**Pull request merged**